src/components/publishing/fixtures/components.ts

```diff
@@ -79,6 +79,17 @@ export const Authors = [
   },
 ]
 
```

Comment:
For the canvas we need the unit fields to be slightly different -- here is what I've been using, let me know if this looks ok to you?
https://github.com/eessex/reaction/blob/display-canvas/src/components/publishing/fixtures/components.ts#L86

Reply:
This makes sense for the Direct layout but make sure it's flexible enough to support the other layouts -- seems like we'd want to rename this to `UnitCanvasDirect` and also include a `layout` field?

Reply:
Yeah, I haven't looked fully into the other layouts yet.  We will need some kind of image-array too.  But they all are in the same database, so we should match the fields.  Specifically brought this up because of the callout link.  It is inline html in the body in this PR, but I think makes sense to separate it out for all units.

Reply:
And yeah - layout field makes sense!

Reply:
I see what you mean with the inline html. My assumption was that this link could exist anywhere in the body which is why it's currently inline. I'll ask Owen if this link always exists at the end and if it's meant to be the same kind of link in Canvas.

Reply:
Ok so I confirmed with Owen that we should keep it inline since we anticipate clients wanting to have that flexibility. The canvas will be the same, so tag-wise, it will be similar to `postscript` where we have some basic functionality like `<p>` and `<a>`

Reply:
Hm in the canvas I was told the entire block is a link this morning, so maybe the broken out link field is specifically for that layout.

Reply:
I think there's some confusion with the design - lets get together with Owen and discuss.